cmd/av/main.go

```diff
@@ -40,6 +43,10 @@ func init() {
 		&rootFlags.Debug, "debug", false,
 		"enable verbose debug logging",
 	)
+	rootCmd.PersistentFlags().StringVarP(
```

Comment:
From the reference it seems like you have to provide:
Directory, "argument_name", "short_code", "", "description"). E.g. from Cobra:
https://cobra.dev/

Am I looking at the wrong thing?
```
StringVarP(&Source, "source", "s", "", "Source directory to read from")
```

Reply:
Also seems like an odd thing to ask for, this should be just part of some config file?

Reply:
I wanted it match `git -C ...` which runs Git in another directory than the working directory. The reason it's needed at all is really just during development. I don't want to run experimental Git commands in this repo to test it, but you have to be in the `av` directory for `go run` to work.

So it's usually:
```
$ pwd
~/Aviator/av

$ go run ./cmd/av -C ~/Aviator/test-repo stack branch
...
```